projboard/tests/test_article.py

```diff
@@ -92,12 +54,11 @@ def test_search_objects(self, create_articles, create_subjects, create_user):
         # search by subject
         assert article_list[1] in Article.search_by_subject(subject1)
 
-    @pytest.mark.django_db
-    def test_search_objects_none_exists(self, create_articles, create_subjects, create_user):
+    def test_search_objects_none_exists(self, articles, subjects, users):
```

Comment:
consider this 
test_search_objects_none_exists -> test_search_non_exist_keys

Reply:
Consider using enums
from enum import Enum
class DataType(Enum):
     EXIST, NON_EXIST = range(0,2)
     
 Then you can call  list like that
 users[DataType.NON_EXIST.value]